examples/guided.py

```diff
@@ -212,13 +212,8 @@ def ask_user_questions():
 
 	# Ask about audio server selection if one is not already set
 	if not archinstall.arguments.get('audio', None):
-		# only ask for audio server selection on a desktop profile
-		if str(archinstall.arguments['profile']) == 'Profile(desktop)':
-			archinstall.arguments['audio'] = archinstall.ask_for_audio_selection()
-		else:
-			# packages installed by a profile may depend on audio and something may get installed anyways, not much we can do about that.
-			# we will not try to remove packages post-installation to not have audio, as that may cause multiple issues
-			archinstall.arguments['audio'] = None
+		# The argument to ask_for_audio_selection lets the library know if it's a desktop profile
+		archinstall.ask_for_audio_selection(is_desktop_profile(archinstall.arguments['profile']))
```

Comment:
We're not interested in the return value?
I assume this would be the case:
```python
archinstall.arguments['audio'] = archinstall.ask_for_audio_selection(is_desktop_profile(archinstall.arguments['profile']))
```
Or why is `ask_for_audio_selection()` returning a value?

Reply:
Nice catch, fixed.